### PR TITLE
refactor: Use core to decrypt assets

### PR DESCRIPTION
**Pull request by atomrc**
## Description

The core already implements encryption and decryption of assets. We should use the methods inside of the core instead of the deprecated methods from the webapp. 

This will also unlock using node > 18 to run tests and CI

see https://github.com/wireapp/wire-web-packages/pull/6110

## Checklist

- [x] PR has been self reviewed by the author;
- [x] Hard-to-understand areas of the code have been commented;
- [x] If it is a core feature, unit tests have been added;

### CODE REVIEW

File: src/script/assets/AssetRepository.ts [hunk 1]
```diff
@@ -146,11 +137,17 @@ export class AssetRepository {
   }
 
   private loadBuffer(asset: AssetRemoteData) {
-    const request = this.core.service!.asset.downloadAsset(asset.urlData, fraction => {
+    const isEncryptedAsset = !!asset.otrKey && !!asset.sha256;
+    const progressCallback = (fraction: number) => {
       asset.downloadProgress(fraction * 100);
-    });
-    asset.cancelDownload = request.cancel;
```
Comment (V-Gira) [inline src/script/assets/AssetRepository.ts:152]: Is this still being handled?

**Review by PatrykBuniX — APPROVED** [review]

File: src/script/assets/AssetRepository.ts (hunk 1 above)
Comment (atomrc) [reply src/script/assets/AssetRepository.ts:152]: Good catch, I think that's an oversight on my end 👁️

Comment (atomrc) [reply src/script/assets/AssetRepository.ts:152]: @V-Gira fixed with https://github.com/wireapp/wire-webapp/pull/17164/commits/fd0a464096143027d3dcac91da5b4218e9a2885d

**Review by V-Gira — APPROVED** [review]

File: src/script/assets/AssetRepository.ts (hunk 1 above)
Comment (V-Gira) [reply src/script/assets/AssetRepository.ts:152]: nice :)

**Pull request merged**